main.sh

```diff
@@ -3,14 +3,7 @@ set -e
 set -o pipefail
 
```

Comment:
You can install dependencies with apt-get (not apt) here.
Be sure to run `sudo apt-get update` first, and sort packages alphabetically.